### PR TITLE
[SPARK-8596][EC2] Added port for Rstudio

### PR DESCRIPTION
**Pull request by koaning**
This would otherwise need to be set manually by R users in AWS.

https://issues.apache.org/jira/browse/SPARK-8596

### CODE REVIEW

**Comment by shivaram** [comment]
Jenkins, add to whitelist

**Comment by shivaram** [comment]
Jenkins, ok to test

File: ec2/spark_ec2.py [hunk 1]
```diff
@@ -505,6 +505,7 @@ def launch_cluster(conn, opts, cluster_name):
         master_group.authorize('tcp', 50070, 50070, authorized_address)
         master_group.authorize('tcp', 60070, 60070, authorized_address)
         master_group.authorize('tcp', 4040, 4045, authorized_address)
+        master_group.authorize('tcp', 8787, 8787, authorized_address)
```
Comment (shivaram) [inline ec2/spark_ec2.py]: Really minor but could you add a comment on the line before saying this is for RStudio ? Might help others figure out why this port is open

**Comment by koaning** [comment]
@shivaram. Comment might make sense (I didn't add because most ports didn't have comments). 

Comment added.

**Comment by koaning** [comment]
just out of curiosity. what is making this test fail?

**Comment by shivaram** [comment]
The tests are not actually failing. The message from SparkQA is the one to use (it says passes all tests). There is a bug in the AmplabJenkins messages that @JoshRosen is looking into.

LGTM. Thanks @koaning for the patch. Merging this

**Pull request closed without merging**